src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -144,9 +142,10 @@
     <Link>
       <SubSystem>Windows</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
-      <AdditionalDependencies>kernel32.lib;user32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;ahadmin.lib;ws2_32.lib;iphlpapi.lib;version.lib</AdditionalDependencies>
+      <AdditionalDependencies>kernel32.lib;user32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;ahadmin.lib;ws2_32.lib;iphlpapi.lib;version.lib;$(LibNetHostPath)\libnethost.lib</AdditionalDependencies>
```

Comment:
Is the $(LibNetHostPath) needed here if it's already in the AdditionalLibraryDirectories?

Reply:
(and same elsewhere)

Reply:
Preference between just using the absolute lib path instead of setting the Directory?  I think I'd rather just nuke the additional library directory changes personally